packages/front-end/pages/metrics.tsx

```diff
@@ -253,7 +254,8 @@ const MetricsPage = (): React.ReactElement => {
               </SortableTH>
             )}
             {showArchived && <th>status</th>}
-            {permissions.createMetrics && !hasFileConfig() && <th></th>}
+            {permissions.check("createMetrics", project) &&
```

Comment:
This won't quite work.  This column is for duplicating metrics.  When you duplicate a metric, it should carry over the selected projects.  So you don't care about the permission of the project in the left nav, just the permission of the existing metric's projects.  That means some rows of this table might have a duplicate button and some may not.  We can just always show this column (as long as `!hasFileConfig()`) and just leave it empty if there aren't permissions for a specific row.

Reply:
Ok, I've conditionally added the button per each metric's projects[] intersection with the user's permissions